From 75fb2a0bd27ce799801bcbae27ff77eba63b967d Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:40:22 -0400
Subject: [ethernet] i40e: Minor comment changes

Message-id: <1409658042-22689-262-git-send-email-sassmann@redhat.com>
Patchwork-id: 90448
O-Subject: [RHEL7.1 PATCH 261/281] i40e: Minor comment changes
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Akeem G Abodunrin <akeem.g.abodunrin@intel.com>

Fixes comment for reset reason

Change-ID: I6fda4fa292255e6eb0f874502b4d38d722149b10
Signed-off-by: Akeem G Abodunrin <akeem.g.abodunrin@intel.com>
Tested-by: Jim Young <jamesx.m.young@intel.com>
Signed-off-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 84a9208d9ee025945dc9ab79f89f9d36cc60f287)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c b/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c
index 8e7a681..0087827 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c
@@ -669,7 +669,7 @@ void i40e_reset_vf(struct i40e_vf *vf, bool flr)
   */
  for (i = 0; i < 100; i++) {
   /* vf reset requires driver to first reset the
-   * vf & than poll the status register to make sure
+   * vf and then poll the status register to make sure
    * that the requested op was completed
    * successfully
    */
-- 
1.7.1